extensions/noteplan-3/src/lib/note-utilities.ts

```diff
@@ -1,14 +1,22 @@
 import { getPreferences } from "./preferences";
-import { parse as parsePath } from "path";
+import { parse as parsePath, join as joinPath } from "path";
 import { homedir } from "os";
 import { formatRelative, parse as parseDate } from "date-fns";
-import { readFileSync } from "fs";
+import { readFileSync, existsSync } from "fs";
 import { sync as find } from "fast-glob";
 import { enGB } from "date-fns/locale";
 import { capitalize } from "lodash";
 import { Icon } from "@raycast/api";
 
-const NOTE_PLAN_URI = `${homedir()}/Library/Containers/co.noteplan.NotePlan3/Data/Library/Application Support/co.noteplan.NotePlan3`;
+function getNotePlan3URI() {
+  const appstoreFlavorPath = `${homedir()}/Library/Containers/co.noteplan.NotePlan3/Data/Library/Application Support/co.noteplan.NotePlan3`;
+  const setappFlavorPath = `${homedir()}/Library/Containers/co.noteplan.NotePlan-setapp/Data/Library/Application Support/co.noteplan.NotePlan-setapp`;
+
+  const useSetappVer: boolean = existsSync(joinPath(setappFlavorPath));
+
+  return useSetappVer ? setappFlavorPath : appstoreFlavorPath;
```

Comment:
Is there an option where a user installed the app through `SetApp` in the past and since then moved to the `AppStore`?

Reply:
I guess this may happen. I also guess it's usually otherwise, because there is no difference in features/functionality between them.
In any case, we can only reliably handle this via some configuration option where user explicitly tells us which version they use. Would you like to add it?

Reply:
Whoops, sorry, missed this notification.
Yes, I think we should add this configuration. 
As much as I don't like adding the need for user intervention, it's better than implicit behaviour without any other way to intervene.

Reply:
Added configuration: now user can select flavour using dropdown, which is set to "AppStore" flavour by default
Please check )